src/lib/style/components/facet_nav/_facet-nav.scss

```diff
@@ -2,43 +2,32 @@
 
 .FacetNav {
   padding-top: 5px;
-  padding-bottom: 5px;
   border: 1px solid SRC.$gray-dark;
 
   &__row {
     background-color: #fff;
 
-    div[class*=' col-'] {
-      height: 200px;
-    }
-  }
-
-  &__showMoreContainer {
     display: flex;
-    justify-content: flex-end;
-    padding-right: 15px;
-    padding-bottom: 10px;
-    &__showMore {
-      margin-right: 15px;
-      margin-top: 15px;
-    }
+    flex-wrap: wrap;
+    justify-content: space-between;
 
-    .FacetNav__showMore:focus {
-      // somehow &__showMore:focus doesn't work
-      outline-color: SRC.$primary-action-color;
+    > div {
+      margin: 5px;
+      flex-grow: 0;
+      flex-basis: 48%;
     }
   }
-}
-
-.container-full-width .FacetNav.less{
-  .FacetNav {
-    &__row {
-      display: flex;
-      flex-wrap: wrap;
 
-      > div {
-        flex: 0 0 380px;
-      }
```

Comment:
Always using flex, so I don't think we need this anymore

Reply:
Ok